ui/src/app/cohort-search/selection-list/selection-list.component.tsx

```diff
@@ -485,16 +443,23 @@ export const SelectionList = fp.flow(withCurrentCohortCriteria(), withCurrentCoh
             <h3 style={{...styles.sectionTitle, marginTop: 0}}>Add selected criteria to cohort</h3>
             <div style={{paddingTop: '0.5rem', position: 'relative'}}>
               <div style={styles.selectionContainer}>
-                {this.renderCriteria()}
+                {!!criteria && criteria.map((selection, s) => <SelectionInfo key={s}
+                  index={s}
+                  selection={selection}
+                  removeSelection={() => this.removeCriteria(selection)}/>
+                )}
                 {this.showModifierButton && <div style={{paddingLeft: '0.6rem'}}>
-                  <Button type='secondaryOnDarkBackground' style={styles.modifierButton}
+                  <Button type='secondaryLight' style={styles.modifierButton}
                           onClick={() => this.setState({showModifiersSlide: true})}>
-                    {modifierButtonText}
+                    {cohortContext.item.modifiers.length > 0
```

Comment:
can cohortContext or cohortContext item be null?

Reply:
There are at least a couple layers that should prevent that. 
- In `help-sidebar.tsx` it checks that `currentCohortSearchContextStore` has a value set before rendering `SelectionList`
- The `showModifiersButton` condition above checks that the `criteria` prop has a value and length, which is only true if `cohortContext` is set.
- Anytime we set a value for `cohortContext`, we include a value for the `item` property, so that should never be `null`